<?xml version="1.0"?>
<documentation title="Some sniff docs" phpVersion="7.3" ref="PHPCompatibility">
    <standard>
    <![CDATA[
      This is an example of the correct documentation example.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: A valid code example">
        <![CDATA[
$number = 3;
if(3 <em>===</em> $number) {
    echo 'Number equals three.';
}
        ]]>
        </code>
        <code title="Invalid: An invalid code example">
        <![CDATA[
$number = 3;
if(3 <em>=</em> $number) {
    echo 'Number equals three.';
}
        ]]>
        </code>
    </code_comparison>
</documentation>
